src/main/kotlin/com/manimdsl/Compiler.kt

```diff
@@ -21,7 +21,14 @@ private fun compile(filename: String) {
     }
 
     val abstractSyntaxTree = parser.convertToAst(program)
-    println(abstractSyntaxTree)
+
+    val executor = ASTExecutor(abstractSyntaxTree)
+
+    do {
+        val state = executor.executeNextStatement()
+        println(state)
```

Comment:
Remove debugging print

Reply:
Leaving it in like the abstractSyntaxTree print for the next step to see how the execution states are to generate the IR from

Reply:
I've added a TODO above it so that it's clear it will be replaced with something else